modules/client/js/src/main/scala/com/azavea/stac4s/api/client/SearchFilters.scala

```diff
@@ -0,0 +1,51 @@
+package com.azavea.stac4s.api.client
```

Comment:
What do you think about their being an `api` module that hold the relevant datamodels? I think it might be a better option than maintaining two copies of the API datamodels and would also help us with the `tapir` / Schema problems if they lived in one place. If you agree it's a good idea I'll open an issue.

Reply:
@jisantuc hm, we can try to handle that; the thing is that we can have the same _models_ but different _codecs_. A good example of it is a `TemporalExtent`: STAC API expects it to be a string and the static STAC requires it to be a JSON object.

Reply:
Ooooh I got what you're talking about. Yes! a separate API module with models sounds good to me. If Franklin and this client would share it this would be awesome! 

That is a great idea to open an issue! // @jisantuc

Reply:
#199